news/2020-01-03-monthly-beatmapping-contests-return.md

```diff
@@ -0,0 +1,135 @@
+---
+layout: post
+title: "Monthly Beatmapping Contests Return"
+date: 2020-01-03 22:00:00 +0000
+---
+
+It's time for a new way to flaunt your beatmapping capabilities. Find out what it takes to become osu!'s next Elite Mapper!
+
+![monthly beatmapping contest banner](/wiki/shared/news/banners/monthly-beatmapping-contest.png)
+
+Beatmapping contests have been far and few between throughout the last couple of years, but that's about to change. The Monthly Beatmapping Contest (MBC) is back with a new format! Participants must adapt to different mapping limitations each month and be evaluated by an unconventional grading system to earn some fancy rewards!
+
+## Timeline
+
+The timeline for each month's contest is as follows:
+- [**Beatmapping (3 weeks):**](#beatmapping) Contest is announced and beatmappers create their entries
+- [**Screening (1 week):**](#screening) Entries are evaluated by experienced beatmappers
+- [**Voting (2 weeks):**](#voting) Top 5 entries from screening are published for the osu! community to vote on
+- [**Rewards:**](#rewards) Winners are announced and prizes are redeemed
+
+"But that's more than one month!", you may be thinking. Each month's contest overlaps, so when a contest is announced at the start of a month, you'll receive access to the voting page for the previous month's contest. Halfway through the month you'll also find out the winner from the previous month's contest (and recieve a reminder of the ongoing contest's submission deadline!).
+
+## <a id="beatmapping"></a>Beatmapping
+
+### <a id="limitation"></a>Limitation
+
+Creating a high quality beatmap is a great goal to strive for, but let's be honest, *that makes for a boring competition.* 
+
+The Monthly Beatmapping Contest requires beatmappers to adhere to a mapping limitation each month. This limitation can be anything from a specific circle size to a limit on star rating.
+
+This month's limitation: **all slider shapes must be straight lines.** No curved sliders or sliders using red anchors are allowed!
```

Comment:
I think adding an image of examples of allowed/disallowed sliders would work well here to clearly illustrate the requirement visually. Also, are the maps required to use sliders or could someone go circles only? Circles only would technically fit the "no curved/angled sliders" requirement if there are no sliders.

Reply:
means what it says, I don't see any room for confusion here